src/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,165 @@
+import PropTypes from 'prop-types'
+import Link from 'gatsby-link'
+import './pagination.css'
+
+function Pagination({ pageContext, type, path }) {
+  let { currentPage, totalPages, minPage, maxPage, formData, formResponse } = pageContext
+  let eachPage = 0
+
+  const apiPageChange = value => {
+    if (value !== currentPage) {
+      return formData(formResponse, value)
+    }
+  }
+
+  const maxPageFn = clicked => {
+    if (clicked === 'More') {
+      if (maxPage + 10 > totalPages) {
+        return totalPages
+      } else {
+        return maxPage + 10
+      }
+    }
+    return maxPage + clicked
+  }
+
+  const direction = value => {
+    const clicked = value.target.innerText === 'More' ? 10 : -10
+    pageContext.setPageContext({
+      currentPage: currentPage,
+      totalPages: totalPages,
+      minPage: minPage + clicked,
+      maxPage: maxPageFn(clicked),
+      setPageContext: pageContext.setPageContext,
+      formResponse: formResponse,
+      formData: formData,
+    })
+  }
+
+  if (type === 'route') {
+    return (
+      <>
```

Comment:
Are these fragments needed?

Reply:
I deleted empty angle brackets that surrounded the div and/or video tags in video.js and pagination.js